data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
> But, what do you mean by "ynh_webpath_register already checks that the url is available"?
register-url uses url-available before?

Yes ([here](https://github.com/YunoHost/yunohost/blob/testing/src/yunohost/app.py#L1110-L1113))

> I think when you simply check the path, you should use the real path like it's stored or defined.

Uh I don't know why you want to use the "real" path when checking availability and not when registering :/ (with or without calling normalize, they're the same anyway, they're just not necessarily formatted the same way ?). Anyway they're renormalized inside the python code so that doesn't really matter ... In fact we should probably normalize the path received from the user/manifest before passing it to the install script.